memo/program/src/lib.rs

```diff
@@ -37,3 +37,19 @@ pub fn build_memo(memo: &[u8], signer_pubkeys: &[&Pubkey]) -> Instruction {
         data: memo.to_vec(),
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_build_memo() {
+        let program_id = Pubkey::new(&[0; 32]);
```

Comment:
Why do you call this program_id? The list of pubkeys passed to the memo program are signers, not necessarily related to any program id. I think this will be more helpful if it's renamed.

Reply:
Also:
```suggestion
        let program_id = Pubkey::new_unique();
```